akka-actor-tests/src/test/scala/akka/pattern/CircuitBreakerStressSpec.scala

```diff
@@ -43,11 +44,14 @@ object CircuitBreakerStressSpec {
       case Failure(ex: CircuitBreakerOpenException) ⇒
         circCount += 1
         breaker.withCircuitBreaker(job).pipeTo(self)
+      case Failure(_: TimeoutException) ⇒
+        timeoutCount += 1
+        breaker.withCircuitBreaker(job).pipeTo(self)
       case Failure(_) ⇒
         failCount += 1
         breaker.withCircuitBreaker(job).pipeTo(self)
       case GetResult ⇒
-        sender() ! Result(doneCount, failCount, circCount)
+        sender() ! Result(doneCount, timeoutCount, failCount, circCount)
```

Comment:
where is this calculated result used? I see it being calculated and reported, but I don't see code that does anything with it.